python/simulation/fake_raft.py

```diff
@@ -0,0 +1,509 @@
+"""
```

Comment:
Aside from the whitespace complaints, some of the things we might want to address are a matter of taste.  I tend to not care very much about things like making constants all caps, as well as invalid function and class names.  For my coding style, I rarely get pylint complaints along those lines. On the other hand, I like to adhere to the 80 column rule whenever possible, since I find it easier to read 80 columns rather than scanning across ~100 columns.  If a column is slightly longer than 80, it wraps in my emacs window, which is a pain.